### ISSUE TITLE
create_ap (cli) does not save the configuration

### ISSUE THREAD

**Issue by sebelk** [lakinduakash/linux-wifi-hotspot#176]
create_ap (cli, version 0..4.6) does not save the configuration.  AFAIK it generates only temporary files. (please correct me if I'm wrong)

`systemctl enable create_ap`  doesn't it ether...:

![imagen](https://user-images.githubusercontent.com/4109256/127776476-6276534a-de83-4cbc-9e3f-88d10c9c1366.png)

It would be nice that either create_ap or via a sistemctl verb generates a persistent configuration.

Thanks in advance.

**Comment by lakinduakash** [comment]
related to #165

**Comment by pcrockett** [comment]
You are correct, the CLI only generates temporary files. However there's a handy parameter you can pass to the CLI that will generate a more permanent configuration file for you. Here's an example command that I ran for my hotspot:

```bash
create_ap --no-virt --no-haveged \
    --mkconfig ./create_ap.conf \
    -m bridge wlan0 end0 ThisIsATest cheeseNBacon
```

This generated a config file at `./create_ap.conf`. I moved it to `/etc/create_ap.conf`, which is where the `create_ap` service reads its configuration from. Run `systemctl restart create_ap` and you should be good to go.
